### PR TITLE
2 71a tests

### PR DESCRIPTION
**Pull request by sonalk0209**
# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by sonalk0209** [comment]
[collect_pr1589.log](https://github.com/Seagate/cortx-test/files/8771253/collect_pr1589.log)
collection output

**Comment by sonalk0209** [comment]
Test logs for both test-42271 and test-42272 are attached in https://jts.seagate.com/browse/CORTX-31387

**Review by kachhwahadivya — APPROVED** [review]

**Review by aditipande96 — APPROVED** [review]

File: tests/csm/rest/test_iam_users.py [hunk 1]
```diff
@@ -4541,25 +4540,103 @@ def test_42270(self):
...
         self.log.info("User list from GET response: %s", get_user_list)
         assert_utils.assert_equals(counter, self.csm_conf["common"]["num_users"],
                                    "Did not get all users")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42271')
+    def test_42271(self):
```
Comment (ravindrachoudhari) [inline tests/csm/rest/test_iam_users.py:4571]: Can we have test name instead of just test ID

File: tests/csm/rest/test_iam_users.py [hunk 2]
```diff
@@ -4541,25 +4540,103 @@ def test_42270(self):
...
+            get_user_list = resp_dict["users"]
+            assert_utils.assert_not_in(user_id, get_user_list, "deleted user still found in list")
+            self.log.info("Deleted user %s is not listed in users list: %s", user_id, get_user_list)
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42272')
+    def test_42272(self):
```
Comment (ravindrachoudhari) [inline tests/csm/rest/test_iam_users.py:4611]: same as above

File: tests/csm/rest/test_iam_users.py (hunk 1 above)
Comment (kachhwahadivya) [reply tests/csm/rest/test_iam_users.py:4571]: In CSM we have been using IDs only. Easier to run if there is CI/CD failure..

Comment (ravindrachoudhari) [reply tests/csm/rest/test_iam_users.py:4571]: ok

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**